ward/suite.py

```diff
@@ -15,9 +15,16 @@ class Suite:
     cache: FixtureCache = field(default_factory=FixtureCache)
 
     @property
-    def num_tests(self):
+    def num_tests(self) -> int:
         return len(self.tests)
 
+    @property
+    def num_tests_with_parameterization(self) -> int:
+        return sum(
+            test.find_number_of_instances() if test.is_parameterised else 1
```

Comment:
Not sure if the if/else is needed here. Hopefully `find_number_of_instances` returns `1` if a test is not parameterised